README.rst

```diff
@@ -24,6 +38,17 @@ Install
 
     pip install pytorch-datastream
 
+Install from source
```

Comment:
This section title could be confusing to people. We are actually describing how to patch for 3.6.

I think it's important that "usage" is very high up to make it easier to get into the lib. Installing from source and patching is very low priority information